locales-src/newtab.ftl

```diff
@@ -51,3 +51,43 @@ newtab-confirm-delete-history-p1 = Are you sure you want to delete every instanc
 newtab-confirm-delete-history-p2 = This action cannot be undone.
 
 newtab-topsites-add-search-engine = Add Search Engine
+
+## Top Site - Action Tooltips.
+# Tooltip to open the context menu for a given page.
+newtab-topsites-tooltip =
+    .title = Open menu
+# Tooltip on an empty topsite box to open the New Top Site dialog.
+newtab-topsites-placeholder-tooltip =
+    .title = Edit this site
+
+## Context Menu: These strings are displayed in a context menu and are meant as a call to action for a given page.
+newtab-menu-edit-topsites = Edit
+newtab-menu-open-new-window = Open in a New Window
+newtab-menu-open-new-private-window = Open in a New Private Window
+newtab-menu-dismiss = Dismiss
+newtab-menu-pin = Pin
+newtab-menu-unpin = Unpin
+newtab-menu-delete-history = Delete from History
+newtab-menu-save-to-pocket = Save to Pocket
+newtab-menu-delete-pocket = Delete from Pocket
+newtab-menu-archive-pocket = Archive in Pocket
```

Comment:
I wonder if we should convert these "Pocket" to `{-pocket-brand-name}` @flodolo ?

https://searchfox.org/mozilla-central/source/browser/locales/en-US/browser/branding/brandings.ftl#14

Reply:
Definitely yes. Ideally, in the existing brandings.ftl file.

Reply:
Sorry, what do you mean "in the existing brandings.ftl" ? These whole strings should move to brandings.ftl instead of only using a pocket placeable?

```ini
# brandings.ftl
## Pocket actions

pocket-save = Save to { -pocket-brand-name }
…
```

vs

```ini
# newtab.ftl
## Context menu

newtab-menu-save-to-pocket = Save to { -pocket-brand-name }
```

Reply:
if we convert [these strings](https://searchfox.org/mozilla-central/source/browser/locales/en-US/chrome/browser/activity-stream/newtab.properties#44) to use placeable, how will it be handled in [migration recipe](https://github.com/mozilla/activity-stream/pull/5104/files#diff-fdfb86f4857398d845489e1c5c373f9aR108)?

Reply:
https://firefox-source-docs.mozilla.org/intl/l10n/l10n/fluent_migrations.html#replacing-content-in-legacy-strings

Reply:
> Sorry, what do you mean "in the existing brandings.ftl" ? These whole strings should move to brandings.ftl instead of only using a pocket placeable?

I meant to have the "Pocket" brand in there, but I forgot we already added it. So, simply replace Pocket with the term reference.